frontend/src/pages/CourseEditor/LessonsStep/LessonItem/styled.ts

```diff
@@ -0,0 +1,84 @@
+import { styled, Box, Input } from '@mui/material';
+
+export const LessonItemWrapper = styled(Box)({
+  margin: '32px 0 50px 0',
+});
+
+export const LessonItemTitle = styled('p')({
+  margin: '0',
+  fontWeight: '400',
+  fontSize: '24px',
+  lineHeight: '31px',
+  letterSpacing: 1,
+  color: '#2C2525',
+});
+
+export const LessonInput = styled(Box)({
+  margin: '30px 0',
+});
+
+export const InputSelect = styled('select')({
+  width: '350px',
+  height: '53px',
+  paddingLeft: '12px',
+  fontWight: 400,
+  fontSize: '18px',
+  lineHeight: '21px',
+  color: '#000',
+  backgroundColor: '#FFF',
+  border: '1px solid',
+  borderColor: 'rgba(0, 0, 0, 0.2)',
+  boxSizing: 'border-box',
+  borderRadius: '5px',
+});
+
+export const SelectItem = styled('option')({});
```

Comment:
please remove an empty component

Reply:
i use it in component 
![image](https://user-images.githubusercontent.com/44974457/167405525-1ca17137-519d-4f3e-9f51-cbd320d94903.png)
![image](https://user-images.githubusercontent.com/44974457/167405579-e75ad232-1a57-48c8-a49d-ee08d3a8cdd0.png)